src/test/groovy/graphql/schema/GraphqlTypeComparatorsTest.groovy

```diff
@@ -38,7 +38,7 @@ class GraphqlTypeComparatorsTest extends Specification {
             
             union XUnion = Foo | Bar
             
-            type Foo implements YInterface, XInterface, ZInterface {
+            type Foo implements YInterface & XInterface & ZInterface {
```

Comment:
This is the case I dont understand ANTLR terms - why can it repeat with more `&` tname entries

Hmmmm - well seems it can so....